app/views/UnloadingGuidanceView.scala.html

```diff
@@ -25,24 +26,35 @@
     config: FrontendAppConfig
 )
 
-@(mrn: MovementReferenceNumber, arrivalId: ArrivalId, messageId: String)(implicit request: Request[_], messages: Messages)
+@(mrn: MovementReferenceNumber, arrivalId: ArrivalId, messageId: String,mode:Mode,  viewModel: UnloadingGuidanceViewModel)(implicit request: Request[_], messages: Messages)
 
 @mainTemplate(
-    title = messages("unloadingGuidance.title"),
+    title = messages(viewModel.title),
     showBackLink = true
 ) {
 
-    @heading(messages("unloadingGuidance.heading"), caption = Some(messages("site.caption", mrn.toString)))
+    @heading(messages(viewModel.heading), caption = Some(messages("site.caption", mrn.toString)))
 
-    <p class="govuk-body">@messages("unloadingGuidance.para", mrn)</p>
+    @viewModel.para1.map { para1Value => <p class="govuk-body">@messages(para1Value)</p> }
 
-    <p class="govuk-body">@messages("unloadingGuidance.pdf.preLinkText")
+    <p class="govuk-body">@messages(s"${viewModel.para2}", mrn.toString)</p>
```

Comment:
I see there's a case where `viewModel.para2` can be an empty string. Just want to check if this is the desired behaviour, or whether `para2` should be an option?

Reply:
Done